index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>html200-adopt-a-dog</title>
+    <link rel="stylesheet" href="./styles/styles.css">
+    <script src="app.js"></script>
+  </head>
+  <body>
+
+    <header class="header-images">
+      <div class="background2Box">
+      <img class="logoHeader"src="./images/adoptadog-logo.svg" alt="Company Logo" id="adoptadog-logo">
+      <h2 class="headertext"> Seattle's premiere dog adoption agency </h2>
+        </div>
+      <a class="checkout" href="./checkout.html">Checkout</a>
+    </header>
+
+    <nav>
+      <ul>
+        <li>
+          <a href="#">Home</a>
+        </li>
+        <li>
+          <a href="./dogs.html">Dogs</a>
+        </li>
+        <li>
+          <a href="./blog.html">Blog</a>
+        </li>
+      </ul>
+    </nav>
+
+    <main id="wrapper">
+      <section class="flex-container">
```

Comment:
This flex-container is a good idea, but you might want to not wrap the intro container in the flex container